games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild

```diff
@@ -3,7 +3,7 @@
 # $Header: /var/cvsroot/gentoo-x86/games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild,v 1.1 2014/10/15 03:50:52 mr_bones_ Exp $
 
 EAPI=5
-inherit eutils qt4-r2 games
+inherit qt4-r2 games
```

Comment:
If the ebuild uses the eclass directly, keep it here! We can never be sure that future versions of `games.eclass` will still inherit `eutils.eclass`.